api/python/quilt3/packages.py

```diff
@@ -1156,7 +1156,7 @@ def top_hash(self):
 
 
     @ApiTelemetry("package.push")
-    def push(self, name, registry=None, dest=None, message=None, selector_fn=lambda logical_key, package_entry: True):
+    def push(self, name, registry=None, dest=None, message=None, selector_fn=None):
```

Comment:
this works in this specific case but has two problems:
1) does not work with future lambda kwargs
2) is not type consistent

can we instead print the lambda function by modifying the gendocs protocol? [here's how to print the lambda](https://stackoverflow.com/questions/30983875/how-do-you-read-a-lambda-function-as-a-string)

Reply:
@akarve

For me this fix looks OK, as I think this is the first time when I see lambda being used as kwarg default.

Also I think it should look OK here too:
```python
help(quilt3.Package.push)

Help on function push in module quilt3.packages:

push(self, name, registry=None, dest=None, message=None, selector_fn=<function Package.<lambda> at 0x7f5d09f6de60>)
    Copies objects to path, then creates a new package that points to those objects.
...
```